test/test_enrolments_view.py

```diff
@@ -14,21 +14,23 @@ class TestEnrolmentsView(PartyTestClient):
 
     @patch("ras_party.views.enrolments_view.respondent_enrolments")
     def test_get_enrolments(self, respondent_enrolments):
```

Comment:
This was already here, but I'm not convinced it's actually testing anything is it? It's simply asserting that the mock object is returned by the test client. Which you'd hope it would be.

Reply:
yeah it's a line coverage one, because the view is light weight and does very little it doesn't look much. But it guards against future iterations changing it and breaking